DHL/JSON/66da0fe7-7a0b-4e36-b685-8f4d6b06607f.json

```diff
@@ -0,0 +1,531 @@
+                              "Id": "6a2708b4-2735-45f2-9cfc-e566935a98c9",
+                              "Nm": "Consent for Greek vat number Category",
+                              "Dsc": "Consent Category Entity",
+                              "isImg": false,
+                              "isExcl": false,
+                              "isMan": false,
+                              "Att": [
+                                {
+                                  "Id": "3b1d09ca-bcce-4572-bf08-5e74f103ae44",
+                                  "Nm": "Consent Category Code",
+                                  "Dsc": "The unique identificaiton of the Consent Purpose",
+                                  "AttTy": "Constant",
+                                  "DflVal": "CSTMS"
+                                },
+                                {
+                                  "Id": "0c36dbe8-4bc1-47dc-8582-8af4dbfc394e",
+                                  "Nm": "Consent Category Name",
+                                  "Dsc": "Consent Category Name",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Customs Consents"
+                                },
+                                {
+                                  "Id": "31e024eb-d270-4074-be28-4ee279a5c544",
+                                  "Nm": "Consent Category Description",
+                                  "Dsc": "Consent Category Desc",
+                                  "AttTy": "Constant",
+                                  "DflVal": "Captures Customs Consents from User"
+                                }
+                              ],
+                              "Rul": [
+                                {
+                                  "RulId": "2c228c8a-5ec3-4cea-922a-1e9348d47bbd",
+                                  "Nm": "Rule condition to Greek vat number",
+                                  "Cond": "**.Att[Id='6a2f893d-8c39-42ed-9b9e-450d21092907'].Val='Opt1'",
+                                  "AttrMtdt": [
+                                    {
+                                      "isMan": true
+                                    }
+                                  ]
+                                }
+                              ]
+                            },
+                            {
+                              "Id": "4829cfe1-fcbb-48d2-9344-d8468de1f41d",
+                              "Nm": "Greek Vat number - Radio button",
+                              "Dsc": "Greek Vat number Radio button desc",
+                              "isImg": false,
+                              "isExcl": false,
+                              "isMan": false,
+                              "Att": [
+                                {
+                                  "Id": "0da3ad86-cc40-44ff-9ad4-60780be50fc8",
+                                  "Nm": "Do you have a Greek VAT number?",
```

Comment:
update nm to -> Greek VAT number Confirmation

Reply:
Done